decompress.jl

```diff
@@ -0,0 +1,44 @@
+using NetCDF,ZfpCompression,BSON
+
+"""
+    decompress_netcdf(oldfile,output)
+
+Decompress the bson file and create a new netcdf file withthe given values and variables.
+
+"""
+
+function decompress_netcdf(comp_file,output)
```

Comment:
What is `output`, the name of the output file? 

This should be included in the documentation. Alternatively, it would probably be better to infer the output file name based on `comp_file`, kind of like the reverse of what you do in the compression function.